src/after.ts

```diff
@@ -1,4 +1,4 @@
-import after = require('lodash/after');
```

Comment:
The problem is this will import the entire library for anybody who is not using tree shaking.